LayoutKit.playground/Sources/UIControl+Handler.swift

```diff
@@ -8,6 +8,7 @@ extension UIControl {
         let target = CocoaTarget<UIControl>(handler)
         objc_setAssociatedObject(self, &controlHandlerKey, target, .OBJC_ASSOCIATION_RETAIN)
 
+        self.removeTarget(target, action: #selector(target.sendNext), for: controlEvents)
```

Comment:
If I'm reading the code correctly, since target is a brand new object instance, it definitely hasn't been registered as a target/action pair on the control and this call won't do anything.

I suspect that the first parameter would have to be the object returned from `objc_getAssociatedObject(self, &controlHandlerKey)` (before the setAssociatedObject call) so that we get a previous CocoaTarget instance if there was one.